data/initrd/initrd.file_list

```diff
@@ -450,7 +453,7 @@ nscd:
 
 # we just want the user & group entries
 rpcbind:
-  /var/lib
+  E groupadd -r nobody || true
```

Comment:
can you comment here *why* we need system group nobody?

Reply:
no we, rpcbind assumes it exists

Reply:
So, can you please add comment here with this explanation, as usually for other cases running prein is enough.